src/core/bridge.ts

```diff
@@ -68,11 +68,12 @@ export abstract class AbstractBridge {
   }
 
   private extractRequest(query: any): { request: Request; queryParams: any; } {
+    const queryParams = Object.assign({}, this.config.queryParams);
     switch (typeof query) {
-      case 'string': return { request: new Query(<string>query).build(), queryParams: {} };
+      case 'string': return { request: new Query(<string>query).build(), queryParams };
       case 'object': return query instanceof Query
-        ? { request: query.build(), queryParams: query.queryParams }
-        : { request: query, queryParams: {} };
+        ? { request: query.build(), queryParams: Object.assign({}, queryParams, query.queryParams) }
```

Comment:
can just be `Object.assign(queryParams, query.queryParams)` as the original `queryParams` from line 71 already builds off of a new empty object